utils/timer_delay.c

```diff
@@ -1,5 +1,5 @@
 /*
- * tim_delay.c
+ * timer_delay.c
  *
  *  Created on: 19-Feb-2021
  *      Author: Sonal Shrivastava
```

Comment:
Can you please add the GPL header to the top of the file (copy over from other files)?

Reply:
yes, done